src/api/informer.rs

```diff
@@ -128,11 +128,33 @@ where
     /// If handling all the events is too time consuming, you probably need a queue.
     pub async fn poll(&self) -> Result<()> {
         trace!("Watching {:?}", self.resource);
-        match self.single_watch().await {
-            Ok((events, newver)) => {
-                *self.version.write().unwrap() = newver;
-                for e in events {
-                    self.events.write().unwrap().push_back(e);
+        let events = self.single_watch().await.map(|stream| stream.boxed());
```

Comment:
would it be hard to have poll return a stream whilst simultaneously storing last resource versions?

ultimately want to expose events to Informer users as they occur as well.

Reply:
Ahh that makes sense 👍, I've updated the informer as such and the informer examples to use the stream returned from `poll`. Now `poll` will fetch the event stream, and just tack on a `map` phase to the stream that will update the resource version as needed as each event streams in.

Reply:
informer's is looking great :+1: